src/style.css

```diff
@@ -1 +1,80 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --main-color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__link {
+  text-decoration: none;
+  color: #000;
+  padding: 20px;
+}
+
+.header__logo-link {
+  max-height: 40px;
+}
+
+.header__list-item {
+  list-style: none;
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)